client/src/components/screens/PatientSubmission/index.jsx

```diff
@@ -289,17 +289,15 @@ const Approval = ({
         <p className="cardDescription">Nullam id dolor id nibh ultricies vehicula ut id elit. Vestibulum id ligula porta felis euismod semper.</p>
         {/* TODO initialValue */}
         <Form.Item className="searchInput searchInput340" label="Médecin résponsable">
-          {getFieldDecorator('practitioner', {
-            rules: [{ required: true, message: 'Veuillez spécifier le nom du médecin responsable' }],
-          })(
-            <AutoComplete
-              classeName="searchInput"
-              placeholder="Recherche par nom ou licence…"
-              dataSource={practitionerOptionsLabels}
-              onSelect={practitionerOptionSelected}
-              onChange={practitionerSearchTermChanged}
-            />,
-          )}
+          <AutoComplete
+            optionLabelProp="text"
+            classeName="searchInput"
+            placeholder="Recherche par nom ou licence…"
```

Comment:
Will the translations be handled in another PR?

Reply:
Exactly